src/pages/receipts/packagesInTransit.js

```diff
@@ -215,12 +215,14 @@ const manifestButton = (allBoxesShippedBySiteAndNotReceived, dataObj, manifestMo
         modalData.groupScannedBy = scannedByArr[index];
         modalData.groupShippedBy = shippedByArr;
         modalData.trackingNumber = trackingNumberArr[index];
-        modalData.siteSpecificLocation = siteSpecificLocationArr[index]
+        modalData.loginSite = groupByLoginSiteCidArr[index];
 
         // Stringify modalData to be parsed later
         button.dataset.modal = JSON.stringify(modalData);
```

Comment:
The data can be stored in state manager.

Reply:
Thank you for the suggestion. I made use of your state manager function and the page loads faster. 

[save packagesInTransitDataObject to state](https://github.com/episphere/biospecimen/pull/527/files#diff-3b1b3cdfa6b45cd7638f994ae8322d8e2b4e0e5b0b3732e282e7489a7711d5ceR104)

[save packagesInTransitDataModal to state](https://github.com/episphere/biospecimen/pull/527/files#diff-3b1b3cdfa6b45cd7638f994ae8322d8e2b4e0e5b0b3732e282e7489a7711d5ceR554)

[retrieve packagesInTransitDataModal to state](https://github.com/episphere/biospecimen/pull/527/files#diff-3b1b3cdfa6b45cd7638f994ae8322d8e2b4e0e5b0b3732e282e7489a7711d5ceR198)